src/GeositeFramework/js/Plugin.js

```diff
@@ -450,6 +450,9 @@ require(['use!Geosite',
                     $('.' + printCssClass).remove();
                     $printSandbox.empty();
 
+                    // add base plugin css
+                    addCss('css/print.css', 'base-plugin-print-css');
```

Comment:
I cut it because after the rules were added dynamically for showing the header, the styles went into a weird state such that the header would always show for plugins. It wouldn't happen if only the plugin output was printed, but once an export brought in the rules they stuck around, even after using jQuery to remove the class.